integrationTests/relayers/slowTests/framework/ethereumHandler.go

```diff
@@ -111,12 +111,16 @@ func (handler *EthereumHandler) DeployContracts(ctx context.Context) {
 	}
 	quorumInt, _ := big.NewInt(0).SetString(handler.Quorum, 10)
 	handler.BridgeAddress = handler.DeployContract(ctx, bridgeABI, bridgeBytecode, ethRelayersAddresses, quorumInt, handler.SafeAddress)
-	handler.BridgeContract, err = contract.NewBridge(handler.BridgeAddress, handler.SimulatedChain)
+	handler.BridgeContract, err = contract.NewBridge(handler.BridgeAddress, handler.SimulatedChain.Client())
 	require.NoError(handler, err)
 
 	// set bridge on safe
 	auth, _ := bind.NewKeyedTransactorWithChainID(handler.OwnerKeys.EthSK, handler.ChainID)
 	tx, err := ethSafeContract.SetBridge(auth, handler.BridgeAddress)
+
+	gasprice, _ := handler.SimulatedChain.Client().SuggestGasPrice(context.Background())
```

Comment:
require.NoError(handler, err) before this line + replace _ with err here?

Reply:
will fix :+1: